mappings/net/minecraft/entity/ai/brain/sensor/Sensor.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS aon net/minecraft/entity/ai/brain/sensor/Sensor
+	FIELD b SENSE_COOLDOWN I
```

Comment:
I don't think this is static final; also since I'm here proposing `interval` rather than cooldown